### PR TITLE
[new release] coin (0.1.2)

### PR DESCRIPTION
**Pull request by dinosaure**
Mapper of KOI8-{U,R} to Unicode

- Project page: <a href="https://github.com/mirage/coin">https://github.com/mirage/coin</a>
- Documentation: <a href="https://mirage.github.io/coin/">https://mirage.github.io/coin/</a>

##### CHANGES:

* Clean the distribution
* Delete provided binary

### CODE REVIEW

File: packages/coin/coin.0.1.2/opam [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+a translation between KOI8-{U,R} and Unicode"""
+
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "ocamlfind"
```
Comment (mseri) [inline packages/coin/coin.0.1.2/opam]: Looks like it is only used by generate.ml
```suggestion
  "ocamlfind" {build}
```

File: packages/coin/coin.0.1.2/opam [hunk 2]
```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+name:         "coin"
```
Comment (mseri) [inline packages/coin/coin.0.1.2/opam]:
```suggestion
```
Opam will override it anyways

**Comment by mseri** [comment]
Two minor changes and it’s good to go! Thanks

**Comment by dinosaure** [comment]
Done, thanks!

**Pull request merged**